e3sm_diags/viewer/aerosol_budget_viewer.py

```diff
@@ -38,33 +29,52 @@ def create_viewer(root_dir, parameters):
         else parameters[0].ref_name
     )
 
-    viewer = OutputViewer(path=root_dir)
+    # Directory for the CSVs and Tables.
+    # Example: aerosol_budgets/Aerosol_table/aerosol_budget/
+    csv_table_dir = os.path.join(root_dir, "..", f"{set_name}")
+
+    # Table data directory -- stores data as CSV and HTML table files, which
+    # populate the viewer's main table.
+    # Example: aerosol_budgets/viewer/table
+    table_dir = os.path.join(root_dir, "table")
+    if not os.path.exists(table_dir):
+        os.mkdir(table_dir)
 
-    # The title of the colums on the webpage.
+    # Create the Viewer object and configure.
+    viewer = OutputViewer(path=root_dir)
     viewer.add_page(display_name, columns=seasons)
-    viewer.add_group("All-Species")
-    viewer.add_row("All-Species")
+    viewer.add_group("All Species")
+    viewer.add_row("All Species")
 
+    # Populate each column with a link to each HTML table file by season.
+    season_html_paths = {}
     for season in seasons:
-        csv_path = os.path.join(
-            csv_table_dir, f"{parameters[0].test_name}-{season}-budget-table.csv"
-        )
+        # Copy the CSV file to the table-data viewer directory.
+        # /aerosol_budgets/Aerosol_table/viewer/../aerosol_budget/v2.LR.historical_0101-ANN-budget-table.csv
+        csv_filename = f"{parameters[0].test_name}-{season}-budget-table.csv"
+        csv_path = os.path.join(csv_table_dir, csv_filename)
         shutil.copy(csv_path, table_dir)
 
-        csv_path = os.path.join(
-            table_dir, f"{parameters[0].test_name}-{season}-budget-table.csv"
-        )
-        print("csv_path", csv_path)
-        html_path = _cvs_to_html(csv_path, season, test_name, ref_name)
+        # Example: /aerosol_budgets/Aerosol_table/viewer/table/v2.LR.historical_0101-ANN-budget-table.csv
+        csv_copy_path = os.path.join(table_dir, csv_filename)
+
+        # Convert the CSV to an HTML table.
+        # Example: aerosol_budgets/Aerosol_table/viewer/table/v2.LR.historical_0101-ANN-budget-table.html
+        html_table_path = _cvs_to_html(csv_copy_path, season, test_name, ref_name)
```

Comment:
I referenced `lat_lon_viewer.py` and its `create_viewer()` because it also calls `_csv_to_html()`. I found out that there is a function to workaround issues with CDP OutputViewer and linking to HTML tables.